content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,675 @@
+```shell
+# Pick one Pod that belongs to the Deployment, and view its logs
+kubectl logs deployments/configmap-volume
+```
+
+You should see an output similar to:
+```
+Found 3 pods, using pod/configmap-volume-76d9c5678f-x5rgj
+Thu Jan  4 14:06:46 UTC 2024 My preferred sport is football
+Thu Jan  4 14:06:56 UTC 2024 My preferred sport is football
+Thu Jan  4 14:07:06 UTC 2024 My preferred sport is football
+Thu Jan  4 14:07:16 UTC 2024 My preferred sport is football
+Thu Jan  4 14:07:26 UTC 2024 My preferred sport is football
+```
+
+Edit the ConfigMap:
+```shell
+kubectl edit configmap sport
+```
+
+In the editor that appears, change the value of key `sport` from `football` to `cricket`. Save your changes.
+The kubectl tool updates the ConfigMap accordingly (if you see an error, try again).
+
+Here's an example of how that manifest could look after you edit it:
+
+```yaml
+apiVersion: v1
+data:
+  sport: cricket
+kind: ConfigMap
+# You can leave the existing metadata as they are.
+# The values you'll see won't exactly match these.
+metadata:
+  creationTimestamp: "2024-01-04T14:05:06Z"
+  name: sport
+  namespace: default
+  resourceVersion: "1743935"
+  uid: 024ee001-fe72-487e-872e-34d6464a8a23
+```
+
+You should see the following output:
+```
+configmap/sport edited
+```
+
+Tail (follow the latest entries in) the logs of one of the pods that belongs to this Deployment:
+```shell
+kubectl logs deployments/configmap-volume --follow
+```
+After few seconds, you should see the log output change as follows:
+```
+Thu Jan  4 14:11:36 UTC 2024 My preferred sport is football
+Thu Jan  4 14:11:46 UTC 2024 My preferred sport is football
+Thu Jan  4 14:11:56 UTC 2024 My preferred sport is football
+Thu Jan  4 14:12:06 UTC 2024 My preferred sport is cricket
+Thu Jan  4 14:12:16 UTC 2024 My preferred sport is cricket
+```
+
+When you have a ConfigMap that is mapped into a running Pod using either a 
+`configMap` volume or a `projected` volume, and you update that ConfigMap, 
+the running Pod sees the update almost immediately.  
+However, your application only sees the change if it is written to either poll for changes, 
+or watch for file updates.  
+An application that loads its configuration once at startup will not notice a change.
+
+
+{{< note >}}
+The total delay from the moment when the ConfigMap is updated to the moment when new keys are projected to the Pod can be as long as kubelet sync period.  
+Also check [Mounted ConfigMaps are updated automatically](/docs/tasks/configure-pod-container/configure-pod-configmap/#mounted-configmaps-are-updated-automatically).
+{{< /note >}}
+
+## Update environment variables of a Pod via a ConfigMap {#rollout-configmap-env}
```

Comment:
I don't think we need this subsection. In a tutorial, we show users how to get something done, step by step. It is weird to read something like "hey, follow me, I'm showing you that something doesn't work."

It is for sure that we need a callout, maybe something like a warning, to tell users that this update cannot apply to environment variables. We can briefly explain why it doesn't work. However, having a dedicated subsection here showing that something doesn't work sounds an overkill.

Reply:
Once a reader follows these steps, the difference between ConfigMap as Volume and as env variable will be crystal clear. I think the section has merit, and reducing it to a callout may be insufficient to demonstrate the difference.

I propose to keep the section, in any case it will not do any harm :)